README.md

```diff
@@ -1,71 +1,86 @@
+<img align="right" src="images/logo.svg" alt="SimpleX logo" height="90">
+
 # SimpleX chat
 
+## Federated, private, secure, decentralized
+
 [![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
 [![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-chat)](https://github.com/simplex-chat/simplex-chat/releases)
 
-## Federated chat - private, secure, decentralized
+SimpleX chat objective: to give you the full control of your chat with free open-source protocol and software of simple to deploy servers and simple to use clients providing complete privacy, security and ownership of your contacts and chat.
+
+SimpleX chat prototype is a _thin_ terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker that uses [SMP protocol](https://github.com/simplex-chat/simplexmq/blob/master/protocol/simplex-messaging.md) and [SMP agent protocol](https://github.com/simplex-chat/simplexmq/blob/master/protocol/agent-protocol.md).
 
 See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
 
-SimpleX chat prototype is a "thin" terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker, that uses [SMP protocol](https://github.com/simplex-chat/simplexmq/tree/master/protocol) and SMP agent protocol.
+## Table of contents
+
+- [Features](#features)
+- [Installation](#installation)
+  - [Download executable chat client](#download-executable-chat-client)
+  - [Build from source](#build-from-source)
+    - [with Docker](#with-docker)
+    - [with Haskell stack](#with-haskell-stack)
+- [Usage](#usage)
+  - [Running the chat client](#running-the-chat-client)
+  - [How to use Simplex](#how-to-use-simplex-chat)
```

Comment:
`with Docker` -> `With Docker` (capital)
`with Haskell stack` -> `With Haskell stack` (also I liked "Using" more - "With" just has a slight connotation that Docker or stack is an add-on for the chat, but w/e)
`How to use Simplex` -> `How to use SimpleX chat`